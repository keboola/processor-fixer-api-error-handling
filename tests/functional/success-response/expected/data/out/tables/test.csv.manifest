{"destination":"out.outputBucket.test","incremental":false}