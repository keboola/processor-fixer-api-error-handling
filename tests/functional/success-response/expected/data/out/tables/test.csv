"success","timestamp","base","date","rates_AED","rates_AFN","rates_ALL","rates_AMD","rates_ANG","rates_AOA","rates_ARS","rates_AUD","rates_AWG","rates_AZN","rates_BAM","rates_BBD","rates_BDT","rates_BGN","rates_BHD","rates_BIF","rates_BMD","rates_BND","rates_BOB","rates_BRL","rates_BSD","rates_BTC","rates_BTN","rates_BWP","rates_BYN","rates_BYR","rates_BZD","rates_CAD","rates_CDF","rates_CHF","rates_CLF","rates_CLP","rates_CNY","rates_COP","rates_CRC","rates_CUC","rates_CUP","rates_CVE","rates_CZK","rates_DJF","rates_DKK","rates_DOP","rates_DZD","rates_EGP","rates_ERN","rates_ETB","rates_EUR","rates_FJD","rates_FKP","rates_GBP","rates_GEL","rates_GGP","rates_GHS","rates_GIP","rates_GMD","rates_GNF","rates_GTQ","rates_GYD","rates_HKD","rates_HNL","rates_HRK","rates_HTG","rates_HUF","rates_IDR","rates_ILS","rates_IMP","rates_INR","rates_IQD","rates_IRR","rates_ISK","rates_JEP","rates_JMD","rates_JOD","rates_JPY","rates_KES","rates_KGS","rates_KHR","rates_KMF","rates_KPW","rates_KRW","rates_KWD","rates_KYD","rates_KZT","rates_LAK","rates_LBP","rates_LKR","rates_LRD","rates_LSL","rates_LTL","rates_LVL","rates_LYD","rates_MAD","rates_MDL","rates_MGA","rates_MKD","rates_MMK","rates_MNT","rates_MOP","rates_MRO","rates_MUR","rates_MVR","rates_MWK","rates_MXN","rates_MYR","rates_MZN","rates_NAD","rates_NGN","rates_NIO","rates_NOK","rates_NPR","rates_NZD","rates_OMR","rates_PAB","rates_PEN","rates_PGK","rates_PHP","rates_PKR","rates_PLN","rates_PYG","rates_QAR","rates_RON","rates_RSD","rates_RUB","rates_RWF","rates_SAR","rates_SBD","rates_SCR","rates_SDG","rates_SEK","rates_SGD","rates_SHP","rates_SLL","rates_SOS","rates_SRD","rates_STD","rates_SVC","rates_SYP","rates_SZL","rates_THB","rates_TJS","rates_TMT","rates_TND","rates_TOP","rates_TRY","rates_TTD","rates_TWD","rates_TZS","rates_UAH","rates_UGX","rates_USD","rates_UYU","rates_UZS","rates_VND","rates_VUV","rates_WST","rates_XAF","rates_XAG","rates_XAU","rates_XCD","rates_XDR","rates_XOF","rates_XPF","rates_YER","rates_ZAR","rates_ZMK","rates_ZMW","rates_ZWL"
"1","1658749143","USD","2022-07-25","3.673015","89.94647","113.916199","409.277837","1.801762","432.01925","129.693924","1.438145","1.7975","1.702819","1.910513","2.018481","94.422357","1.912599","0.376936","2059.273441","1","1.384642","6.873004","5.498601","0.999668","4.5464672E-5","79.75433","12.606767","2.523932","19600","2.015082","1.286815","2002.500677","0.963735","0.034517","952.429806","6.7464","4465","671.417268","1","26.5","107.709064","23.959811","177.976615","7.27117","54.344938","145.728526","18.945405","15","52.589745","0.976625","2.19355","0.842337","0.82878","2.94008","0.842337","8.272591","0.842337","54.090173","8655.504869","7.732778","209.153647","7.849335","24.590708","7.344898","117.235181","387.901579","14943.55","3.44072","0.842337","79.783015","1459.090376","42350.000153","136.230103","0.842337","152.43719","0.709036","136.490411","118.670287","80.421698","4093.755189","464.625046","899.999992","1310.119585","0.30706","0.83304","477.91198","15052.11335","1511.531361","354.882099","152.49797","15.839945","2.95274","0.60489","4.874575","10.201618","19.304497","4203.815496","60.186768","1850.939399","3144.659992","8.082365","356.999828","45.440802","15.350096","1026.104539","20.408498","4.453501","63.830124","15.840029","425.497258","35.844412","9.84135","127.605642","1.59453","0.384485","0.999668","3.915624","3.522511","55.873969","229.805419","4.601215","6886.453327","3.641027","4.819505","114.709768","58.062506","1032.606572","3.756847","8.159969","13.148475","456.499573","10.15793","1.384849","1.377401","13170.00004","584.497091","23.257501","20697.981008","8.747424","2512.529433","16.762787","36.66899","10.09685","3.51","3.063502","2.34345","17.847098","6.798763","29.897502","2331.341018","36.558141","3846.326668","1","41.803423","10963.84789","23390","118.896025","2.72097","640.751956","0.053606","0.000579","2.70255","0.759966","640.751956","113.149693","250.249783","16.76377","9001.189851","16.320381","321.999592"
